locales/en.json

```diff
@@ -551,7 +551,9 @@
           },
           "metamask": {
             "action": "Pay with Metamask",
```

Comment:
Do we use this does it still make sense to have it as `pay with metamask`?

Reply:
updated now